tests/tests_change_settings.yml

```diff
@@ -0,0 +1,101 @@
+    - name: Ensure required packages are installed
+      package:
+        name: tuned
+        state: present
+
+    - name: Ensure kernel settings profile directory exists
+      file:
+        path: /etc/tuned/kernel_settings
+        state: directory
+        mode: 0755
+
+    - name: Generate a configuration for kernel settings
+      copy:
+        src: tuned/etc/tuned/basic_settings/tuned.conf
+        dest: /etc/tuned/kernel_settings/tuned.conf
+        mode: 0644
+
+    - name: Ensure required services are enabled and started
+      service:
+        name: tuned
+        state: restarted
+        enabled: yes
+
+    - name: apply kernel_settings
+      include_role:
+        name: linux-system-roles.kernel_settings
+      vars:
+        kernel_settings_sysctl:
+          - name: fs.file-max
+            value: 400000
+          - name: kernel.threads-max
+            state: absent
+        kernel_settings_sysfs:
+          - name: /sys/class/net/lo/mtu
+            value: 65000
+        kernel_settings_bootloader_cmdline:
+          - name: spectre_v2
+            value: "on"
+          - name: quiet
+            state: absent
+
+    - name: check with tuned verify
+      command: tuned-adm verify -i
+
+    - name: check sysfs after role runs
+      command: grep 65000 /sys/class/net/lo/mtu
+
+    - name: check sysctl after role runs
+      shell: sysctl -n fs.file-max | grep 400000
+
+    - name: check sysctl after role runs
+      shell: sysctl -n kernel.threads-max | grep -vq 29968
+
+    - name: reboot the machine - see if settings persist after reboot
+      reboot:
+        test_command: tuned-adm active
+
+    - name: check sysctl after reboot
+      shell: sysctl -n fs.file-max | grep 400000
+
+    - name: check sysfs after reboot
+      command: grep 65000 /sys/class/net/lo/mtu
+
+    - name: check cmdline for spectre_v2 after reboot
+      command: grep spectre_v2=on /proc/cmdline
+
+    - name: check cmdline for quiet flag after reboot (should not present)
+      command: grep -vq quiet /proc/cmdline
+
+    - name: check sysctl after reboot
+      shell: sysctl -n kernel.threads-max | grep -vq 29968
+
+    - name: check with tuned verify
+      command: tuned-adm verify -i
+
+    - name: apply role again and remove settings
+      include_role:
+        name: linux-system-roles.kernel_settings
+      vars:
+        kernel_settings_sysctl:
+          - name: fs.file-max
+            value: absent
+          - name: kernel.threads-max
+            state: absent
+        kernel_settings_bootloader_cmdline:
+          - name: spectre_v2
+            state: absent
+          - name: quiet
+            state: absent
+
+    - name: reboot the machine - see if settings persist after reboot
+      reboot:
+        test_command: tuned-adm active
+
+    - name: check after spectre_v2 reboot
+      command: grep -vq spectre_v2 /proc/cmdline
+
+    - name: check sysctl after reboot
+      shell: sysctl -n fs.file-max | grep -vq 400000
```

Comment:
you can also do `command: grep -vq 400000 /proc/sys/fs/file-max` - as you are probably aware, every setting known by `sysctl` is really a file under `/proc/sys`